cfme/fixtures/pytest_selenium.py

```diff
@@ -112,25 +112,37 @@ def elements(o, **kwargs):
 
     Returns: A list of WebElement objects
     """
-    check_visibility = kwargs.pop("check_visibility", True)
-    if hasattr(o, "locate"):
-        els = elements(o.locate(), **kwargs)
-        if check_visibility:
-            return [e for e in els if is_displayed(e)]
-        else:
-            return els
-    elif callable(o):
-        els = elements(o(), **kwargs)
-        if check_visibility:
-            return [e for e in els if is_displayed(e)]
+    if 'tries' in kwargs:
+        tries = kwargs['tries']
+    else:
+        tries = 3
```

Comment:
This has the potential to slow things down by taking the triple the amount of time to time out on a block, what are we introducing it for? If it's a single specific place, we could just use a wait_for with is_displayed?